packages/mdc-elevation/_mixins.scss

```diff
@@ -25,6 +25,23 @@
 @import "@material/theme/variables";
 @import "./variables";
 
+@mixin mdc-elevation-top-level($query: mdc-feature-all()) {
+  $feat-structure: mdc-feature-create-target($query, structure);
+
+  @for $z-value from 0 through 24 {
+    .mdc-elevation--z#{$z-value} {
+      @include mdc-elevation($z-value, $query: $query);
+    }
+  }
+
+  .mdc-elevation-transition {
+    @include mdc-feature-targets($feat-structure) {
+      transition: mdc-elevation-transition-value();
```

Comment:
Wouldn't this be under animation?

Reply:
Whoops, I think I read that as `transform` in my head